packages/syft/proto/core/tensor/tensor.proto

```diff
@@ -13,6 +15,9 @@ message Tensor {
 //  repeated syft.lib.numpy.NumpyProtoArrow arrays = 5;
 //  repeated Tensor tensors = 6;
 //  bool requires_grad = 7;
```

Comment:
MAYBE: We could remove the above repeated comments

Reply:
Yep